css/components/_btn.scss

```diff
@@ -0,0 +1,81 @@
+
+.btn {
+  border: 1px solid transparent;
+  border-radius: .25rem;
+  color: $pure-white;
+  cursor: pointer;
+  display: inline-block;
+  font-weight: 400;
+  padding: 1px;
+  position: relative;
+  text-align: center;
+  user-select: none;
+  vertical-align: middle;
+
+  a {
+    height: 100%;
+    width: 100%;
+  }
+
+  a,
```

Comment:
It's good practice to have some kind of variation for links when they are focused or hovered. This can be either a change of color or the use of `outline` or an underline.